src/App.tsx

```diff
@@ -1,14 +1,31 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { useDispatch } from 'react-redux';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppSelector } from './app/hooks';
+import { getTodos } from './api';
+import { setTodos } from './features/todos';
 
 export const App: React.FC = () => {
+  const dispatch = useDispatch();
+  const selectedTodo = useAppSelector(state => state.currentTodo);
+  const todos = useAppSelector(state => state.todos);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoaded, setIsLoaded] = useState(false);
```
`isLoading` means that in the current moment, loading is in process, but according to cases you use this value, you store `true` if loading is done, so it should be named properly